pkgs/development/python-modules/openai-triton/llvm.nix

```diff
@@ -67,24 +68,33 @@ stdenv.mkDerivation (finalAttrs: {
 
   sourceRoot = "${finalAttrs.src.name}/llvm";
 
-  cmakeFlags = [
-    "-DLLVM_TARGETS_TO_BUILD=X86;AMDGPU;NVPTX"
-    "-DLLVM_ENABLE_PROJECTS=llvm;mlir"
-    "-DLLVM_INSTALL_UTILS=ON"
-  ] ++ lib.optionals (buildDocs || buildMan) [
-    "-DLLVM_INCLUDE_DOCS=ON"
-    "-DMLIR_INCLUDE_DOCS=ON"
-    "-DLLVM_BUILD_DOCS=ON"
-    # "-DLLVM_ENABLE_DOXYGEN=ON" Way too slow, only uses one core
-    "-DLLVM_ENABLE_SPHINX=ON"
-    "-DSPHINX_OUTPUT_HTML=ON"
-    "-DSPHINX_OUTPUT_MAN=ON"
-    "-DSPHINX_WARNINGS_AS_ERRORS=OFF"
-  ] ++ lib.optionals buildTests [
-    "-DLLVM_INCLUDE_TESTS=ON"
-    "-DMLIR_INCLUDE_TESTS=ON"
-    "-DLLVM_BUILD_TESTS=ON"
-  ];
+  cmakeFlags =
+    let
+      # Targets can be found in
+      # https://github.com/llvm/llvm-project/tree/f28c006a5895fc0e329fe15fead81e37457cb1d1/clang/lib/Basic/Targets
+      # NOTE: Unsure of how "host" would function, especially given that we might be cross-compiling.
+      llvmTargets = [ "AMDGPU" "NVPTX" ]
+        ++ lib.optionals stdenv.isAarch64 [ "AArch64" ]
+        ++ lib.optionals stdenv.isx86_64 [ "X86" ];
+    in
+    [
+      "-DLLVM_TARGETS_TO_BUILD=${lib.concatStringsSep ";" llvmTargets}"
+      "-DLLVM_ENABLE_PROJECTS=llvm;mlir"
+      "-DLLVM_INSTALL_UTILS=ON"
+    ] ++ lib.optionals (buildDocs || buildMan) [
```

Comment:
Is there a way to keep the same number of spaces? 😂 I hate how bad github is at rendering this kind of diffs

Reply:
Fixed in latest -- inlined it into the string ;)